src/data/actions/pingStudio.js

```diff
@@ -0,0 +1,19 @@
+export const PING_RESPONSE = 'PING_RESPONSE';
+
+function pingResponse(response) {
```

Comment:
This is a vanilla, non-asynchronous action creator, returning a very simple action (like a message for the queue) containing response status. The `PING_RESPONSE` label is important -- we'll see why later. Docs on actions and action creators available here: http://redux.js.org/docs/basics/Actions.html